src/scenarios/shared/startup.py

```diff
@@ -16,12 +16,14 @@ class StartupWrapper(object):
     Wraps startup.exe, building it if necessary.
     '''
     def __init__(self):
-        payload_startup = os.path.join(helixpayload(), 'Startup')
-        workitem_startup = os.path.join(helixworkitempayload(), 'Startup')
-        if os.path.exists(payload_startup):
-            self._setstartuppath(payload_startup)
-        elif os.path.exists(workitem_startup):
-            self._setstartuppath(workitem_startup)
+        if helixpayload():
+            payload_startup = os.path.join(helixpayload(), 'Startup')
+            if os.path.exists(payload_startup):
+                self._setstartuppath(payload_startup)
+        elif helixworkitempayload():
+            workitem_startup = os.path.join(helixworkitempayload(), 'Startup')
+            if os.path.exists(workitem_startup):
+                self._setstartuppath(workitem_startup)
```

Comment:
Did you change this or did you not merge the previous change into whatever branch this is working off of?

Reply:
Lemme check. I did pull from the upstream master branch before working on it.

Reply:
The current code on master is not checking if the helixpayload or helixworkitempayload is null and breaks on client runs.